### PR TITLE
Fixes #112: Updated 'Get Started' button link in footer

### LINKED ISSUE #112: Fix Get Started link
Currently when click on Get Started button in footer navigating to `/group/new` replace that to new route `/create-group`
Screenshot 2024-10-25 at 01 34 25

### PR DESCRIPTION
**Pull request by InderParmar**
Fixes #112: Updated the "Get Started" button link in the footer from `/group/new` to `/create-group`. Verified the link to ensure it navigates correctly to the new route.

### CODE REVIEW

**Comment by harshmangalam** [comment]
Can you please remove lock file from this PR ?

**Comment by InderParmar** [comment]
Hi @harshmangalam, I have removed the lock files, can you now review, and merge if its good?

File: pnpm-lock.yaml
Comment (harshmangalam) [inline pnpm-lock.yaml:1]: This is still in the PR

Comment (InderParmar) [reply pnpm-lock.yaml:1]: Hi @harshmangalam ,

I’ve removed both package-lock.json and pnpm-lock.yaml as requested. I also made sure to force-sync my branch to ensure all changes are fully reflected in the PR. Could you please review it again to confirm that the lock files are no longer included?

Thank you for your guidance and patience!

Comment (InderParmar) [reply pnpm-lock.yaml:1]: @harshmangalam  can you also please add the "hacktoberfest-accepted" label to this issue.

**Review by harshmangalam — APPROVED** [review]

**Pull request merged**